internal/prompter/prompter_test.go

```diff
@@ -0,0 +1,58 @@
+package prompter
+
+import (
+	"testing"
+
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewReturnsAccessiblePrompter(t *testing.T) {
```

Comment:
I think generally we might want to refer to this thing as `screenreaderFriendlyPrompter` rather than `accessiblePrompter`. It's a clearer link to the env var, and accessibility is a big scope.